docs/package.json

```diff
@@ -4,12 +4,12 @@
   "description": "preact material component's site",
   "scripts": {
     "start": "if-env NODE_ENV=production && npm run -s serve || npm run -s dev",
-    "build": "preact build --no-prerender",
-    "build:prod": "preact build --template template.html --no-prerender && npm run copyAssets",
+    "build": "preact build",
+    "build:prod": "preact build --template template.html && npm run copyAssets",
     "copyAssets": "cp -R images *.png *.json build/",
     "serve": "preact build && preact serve",
     "dev": "preact watch",
-    "test": "npm run prettier",
+    "test": "npm run build:prod && ../node_modules/.bin/mocha ./tests/visdiff.js --timeout 5000 --exit",
```

Comment:
Would be nice to keep `prettier` in the `test` script.

Reply:
done